app/index.js

```diff
@@ -41,7 +42,8 @@ app.on('window-all-closed', () => {
 
 app.on('ready', () => {
   function createWindow (fn) {
-    const cfg = plugins.getDecoratedConfig();
+    let cfg = plugins.getDecoratedConfig();
```

Comment:
Needs to be `let` since it's gets reassigned when the config changes, on [line 86](https://github.com/zeit/hyperterm/pull/405/files#diff-2018087f584c4398b5c3a23fc0e5f9dbR86)